Project1-WeatherRadar/weather-radar-gateway/routes/index.js

```diff
@@ -14,10 +14,12 @@ router.all('/:apiName/:path', (req, res) => {
             method: req.method,
             url: formattedPath,
             headers: req.headers,
-            data: req.body
+            data: req.body,
+            responseType: 'arraybuffer'
         }).then((response) => {
             console.log('API SUCCESS RESPONSE:'+ formattedPath +':'+ response.data);
-            res.send({'Content-Type':'image/gif'},response.data)
+            data = Buffer.from(response.data, 'binary').toString('base64')
+            res.send(data)
```

Comment:
gateway should be generic, we will be handling data format related issue at backend or frontend. Revert this changed to default , lets keep the gateway generic as discussed.

Reply:
Okay. I will revert the changes

Reply:
As discussed, reverted the changes and now the gateway communication will be generic irrespective of the data format.
Please review and merge.